.github/workflows/testpydra.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+
+name: Test Pydra
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        python-version: [3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Update build tools
+      run: |
+        python -m pip install --upgrade pip==20.2.1 setuptools==30.2.1 wheel
+    - name: Install dependencies
+      run: |
+        python -m pip install -r min-requirements.txt
+    - name: Install pydra
+      run: |
+        python -m pip install .[test]
+    - name: Pytest
+      env:
+        IGNORE_TESTS: '*dockertask.py'  # docker tests run in a seperate action for now
```

Comment:
any chances you forgot to add a new workflow to the git repository? I don't see the action with docker tests.

Reply:
I have the docker one in a separate branch!  Not completely done yet - https://github.com/nicolocin/pydra/tree/docker-ga